csr/csr.go

```diff
@@ -216,11 +231,21 @@ func ParseRequest(req *CertificateRequest) (csr, key []byte, err error) {
 			Bytes: key,
 		}
 		key = pem.EncodeToMemory(&block)
+	case ed25519.PrivateKey:
+		key, err = derhelpers.MarshalEd25519PrivateKey(priv)
+		if err != nil {
+
```

Comment:
Is there a line missing here?

Reply:
Oh, yes! I'll add it ;) Thanks for pointing it out!